apps/pptx-worker/features/visualization/generation_pipeline.py

```
@@ -0,0 +1,853 @@
+                    ),
+                    occurred_at=self._now_iso(),
+                    schema_version=task.schema_version,
+                    current_fills=fills,
+                )
+                return _ContentOutcome(
+                    registered_slide=registered_slide,
+                    planned_slide=planned_slide,
+                    status="ready",
+                    current_fills=fills,
+                )
+            except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)